components/Voting/Elo.tsx

```diff
@@ -52,10 +53,17 @@ function EloChoice({ session, variant = 'primary', onChoice }: EloChoiceProps) {
     onChoice(session)
   }
 
+  const abstract = session.Abstract
+  const paragraphs = useMemo(() => abstract.split('\n').filter((para) => !!para), [abstract])
+
   return (
     <StyledEloChoice variant={variant}>
       <StyledSessionTitle>{session.Title}</StyledSessionTitle>
-      <StyledSessionAbstract>{session.Abstract}</StyledSessionAbstract>
+      <StyledSessionAbstract>
+        {paragraphs.map((para) => (
+          <p key={para}>{para}</p>
```

Comment:
Would this look ok using the <Text> component?

Reply:
The `StyledSessionAbstract` component? I'd need to move the `overflowY: auto` stuff onto a parent, but could do

Reply:
That's weird, the `<Text>` part disappeared. I just meant instead of `<p>`

Reply:
I'll swap it over. Looks basically the same: margin between paragraphs is a little bigger (24px vs 8px), colour is slightly different (#292929 vs #333)